pkg/phlaredb/delta.go

```diff
@@ -97,12 +100,21 @@ func copySample(s *schemav1.Sample) *schemav1.Sample {
 }
 
 func isDelta(lbs phlaremodel.Labels) bool {
+	if lbs.Get(phlaremodel.LabelNameDelta) == "false" {
```

Comment:
Nit

It's a bit confusing that you can only use the label to deactivate.

May be if the labels is true then we should return true too and if not a Boolean like empty go to the rest of the function.